testdata/p4_16_errors_outputs/table-entries-decl-order.p4-stderr

```diff
@@ -0,0 +1,12 @@
+../testdata/p4_16_errors/table-entries-decl-order.p4(53): error: Table initializer entries must occur after key definitions (table t_ternary)
+    table t_ternary {
+          ^^^^^^^^^
+../testdata/p4_16_errors/table-entries-decl-order.p4(66): error: Could not find type of Key
```

Comment:
No, i don't want repeated messages, but these are different entries, and each of them gets checked. Are you suggesting to prune the rest of the children if an error is encountered? I thought we were leaning toward reporting as many errors as possible.